ui/i18n/src/main/res/values/strings.xml

```diff
@@ -93,6 +93,7 @@
 
     <!-- Playback history -->
     <string name="clear_history_label">Clear History</string>
+    <string name="date_range_picker_title">Choose a start date and an end date</string>
```

Comment:
I would just use the menu item title there. Makes it shorter and fewer strings to translate :)